WordPress/Classes/ViewRelated/Blog/TimeZoneSelectorViewController.swift

```diff
@@ -0,0 +1,112 @@
+import UIKit
+
+class TimeZoneSelectorViewController: UITableViewController, ImmuTablePresenter {
+
+    /// users current timezone passed by SiteSettingsVC, if empty means manual offset is to be used
+    @objc var usersCurrentTimeZone: String?
```

Comment:
I'd probably rename these to either drop `user` or switch it for `blog`, as it's site specific not user specific. `currentTimeZone`  and `currentManualOffset`? `currentBlogTimeZone` and `currentBlogManualOffset`? Or perhaps `initialTimeZone` and `initialManualOffset` might make most sense, as they're only set on initialization.

On that note, I'd probably change these two properties to be private constants, and have an initializer to set them.